### PR TITLE
Refactoring the assets manifest for Ingress Controller

### PR DESCRIPTION
**Pull request by vladciobancai**
Resolving https://github.com/digitalocean/Kubernetes-Starter-Kit-Developers/issues/97

### CODE REVIEW

File: 03-setup-ingress-controller/nginx.md [hunk 1]
```diff
@@ -272,17 +272,17 @@ Steps to follow:
 3. Then, create the [echo](assets/manifests/echo_deployment.yaml) and [quote](assets/manifests/quote_deployment.yaml) deployments:
```
Comment (v-ctiutiu) [inline 03-setup-ingress-controller/nginx.md:272]: Manifest file path for the `echo` and `quote` deployments is incorrect. Should be:

```
[echo](assets/manifests/nginx/echo_deployment.yaml)
[quote](assets/manifests/nginx/quote_deployment.yaml)
```

File: 03-setup-ingress-controller/nginx.md [hunk 2]
```diff
@@ -388,22 +388,22 @@ DESCRIPTION:
   documentation.
 ```
 
-You can define `multiple rules` for different `hosts` and `paths` in a single `ingress` resource. To keep things organized (and for better visibility), `Starter Kit` tutorial provides two ingress manifests for each host: [echo](assets/manifests/echo_host_nginx.yaml) and [quote](assets/manifests/quote_host_nginx.yaml).
+You can define `multiple rules` for different `hosts` and `paths` in a single `ingress` resource. To keep things organized (and for better visibility), `Starter Kit` tutorial provides two ingress manifests for each host: [echo](assets/manifests/echo_host.yaml) and [quote](assets/manifests/quote_host.yaml).
```
Comment (v-ctiutiu) [inline 03-setup-ingress-controller/nginx.md:391]: Manifest file path for the `echo` and `quote` hosts is incorrect. Should be:

```
[echo](assets/manifests/nginx/echo_host.yaml)
[quote](assets/manifests/nginx/quote_host.yaml)
```

**Review by v-ctiutiu — CHANGES_REQUESTED** [review]

File: 03-setup-ingress-controller/nginx.md (hunk 2 above)
Comment (vladciobancai) [reply 03-setup-ingress-controller/nginx.md]: Applied the fix

**Review by v-ctiutiu — APPROVED** [review]
Looks good to me now.

**Pull request merged**